server/sonar-server/src/main/java/org/sonar/server/component/ws/AppAction.java

```diff
@@ -168,6 +178,7 @@ private void appendComponent(JsonWriter json, ComponentDto component, UserSessio
     if (branch != null) {
       json.prop("branch", branch);
     }
+    //TODO set pullRequest and pullRequestTitle
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fbranch%2Ffirst-class-pull-request&issues=AWGpkpHDWHTwXubXzcEY&open=AWGpkpHDWHTwXubXzcEY)

[ISSUE_KEY]: AWGpkpHDWHTwXubXzcEY